### PR TITLE
docs: added example for awscc_verifiedpermissions_policy_store

### PR DESCRIPTION
**Pull request by quixoticmonk**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

File: examples/resources/awscc_verifiedpermissions_policy_store/verifiedpermissions_policy_store.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...paddr", "required" : false }, "authenticated" : { "type" : "Boolean", "required" : true } } } }, "entityTypes" : { "User" : { "shape" : { "type" : "Record", "attributes" : { "userId" : { "type" : "String" }, "personInformation" : { "type" : "PersonType" } } }, "memberOfTypes" : ["UserGroup"] }, "UserGroup" : { "shape" : { "type" : "Record", "attributes" : {} } }, "Photo" : { "shape" : { "type" : "Record", "attributes" : { "account" : { "type" : "Entity", "name" : "Account", "required" : true }, "private" : { "type" : "Boolean", "required" : true } } }, "memberOfTypes" : ["Album", "Account"] }, "Album" : { "shape" : { "type" : "Record", "attributes" : {} } }, "Account" : { "shape" : { "type" : "Record", "attributes" : {} } } }, "actions" : { "viewPhoto" : { "appliesTo" : { "principalTypes" : ["User", "UserGroup"], "resourceTypes" : ["Photo"], "context" : { "type" : "ContextType" } } }, "createPhoto" : { "appliesTo" : { "principalTypes" : ["User", "UserGroup"], "resourceTypes" : ["Photo"], "context" : { "type" : "ContextType" } } }, "listPhotos" : { "appliesTo" : { "principalTypes" : ["User", "UserGroup"], "resourceTypes" : ["Photo"], "context" : { "type" : "ContextType" } } } } } })
```
Comment (wellsiau-aws) [inline examples/resources/awscc_verifiedpermissions_policy_store/verifiedpermissions_policy_store.tf:7]:
```suggestion
    cedar_json = jsonencode({
      "PhotoApp": {
        "commonTypes": {
          "PersonType": {
            "type": "Record",
            "attributes": {
              "age": {
                "type": "Long"
              },
              "name": {
                "type": "String"
              }
            }
          },
          "ContextType": {
            "type": "Record",
            "attributes": {
              "ip": {
                "type": "Extension",
                "name": "ipaddr",
                "required": false
              },
              "authenticated": {
                "type": "Boolean",
                "required": true
              }
            }
          }
        },
        "entityTypes": {
          "User": {
            "shape": {
              "type": "Record",
              "attributes": {
                "userId": {
                  "type": "String"
                },
                "personInformation": {
                  "type": "PersonType"
                }
              }
            },
            "memberOfTypes": [
              "UserGroup"
            ]
          },
          "UserGroup": {
            "shape": {
              "type": "Record",
              "attributes": {}
            }
          },
          "Photo": {
            "shape": {
              "type": "Record",
              "attributes": {
                "account": {
                  "type": "Entity",
                  "name": "Account",
                  "required": true
                },
                "private": {
                  "type": "Boolean",
                  "required": true
                }
              }
            },
            "memberOfTypes": [
              "Album",
              "Account"
            ]
          },
          "Album": {
            "shape": {
              "type": "Record",
              "attributes": {}
            }
          },
          "Account": {
            "shape": {
              "type": "Record",
              "attributes": {}
            }
          }
        },
        "actions": {
          "viewPhoto": {
            "appliesTo": {
              "principalTypes": [
                "User",
                "UserGroup"
              ],
              "resourceTypes": [
                "Photo"
              ],
              "context": {
                "type": "ContextType"
              }
            }
          },
          "createPhoto": {
            "appliesTo": {
              "principalTypes": [
                "User",
                "UserGroup"
              ],
              "resourceTypes": [
                "Photo"
              ],
              "context": {
                "type": "ContextType"
              }
            }
          },
          "listPhotos": {
            "appliesTo": {
              "principalTypes": [
                "User",
                "UserGroup"
              ],
              "resourceTypes": [
                "Photo"
              ],
              "context": {
                "type": "ContextType"
              }
            }
          }
        }
      }
    })
```

Comment (wellsiau-aws) [reply examples/resources/awscc_verifiedpermissions_policy_store/verifiedpermissions_policy_store.tf:7]: beautify the json schema for readability

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
@quixoticmonk , small nit about the schema, please take a look, thanks !

**Comment by quixoticmonk** [comment]
> @quixoticmonk , small nit about the schema, please take a look, thanks !

Took care of the changes. Merged the changes from latest merge into main as well.

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thank you!

**Comment by marcosentino** [comment]
@quixoticmonk Thank you for the contribution!

**Pull request merged**